lib/rspec/core/notifications.rb

```diff
@@ -199,6 +199,12 @@ def fully_formatted(failure_number, colorizer=::RSpec::Core::Formatters::Console
         @exception_presenter.fully_formatted(failure_number, colorizer)
       end
 
+      # @return [Array] The failure information fully formatted in the way that
```

Comment:
Instead of `[Array]`, how about the more specific `[Array<String>]`?  That's what we've done for the type elsewhere (such as on `FailedExampleNotification#message_lines`).